cli/azd/extensions/registry.json

```diff
@@ -1333,152 +1035,367 @@
               "url": "https://github.com/Azure/azure-dev/releases/download/azd-ext-azure-ai-agents_0.1.2-preview/azure-ai-agents-windows-arm64.zip"
             }
           }
+        }
+      ]
+    },
+    {
+      "id": "azure.ai.finetune",
+      "namespace": "ai.finetuning",
+      "displayName": "Foundry Fine Tuning (Preview)",
+      "description": "Extension for Foundry Fine Tuning. (Preview)",
+      "versions": [
+        {
+          "capabilities": [
+            "custom-commands",
+            "lifecycle-events",
+            "service-target-provider"
+          ],
+          "version": "0.0.2-preview",
+          "usage": "azd ai finetuning \u003ccommand\u003e [options]",
+          "examples": [
+            {
+              "name": "init",
+              "description": "Initialize a new AI fine-tuning project.",
+              "usage": "azd ai finetuning init"
+            },
+            {
+              "name": "deploy",
+              "description": "Deploy AI fine-tuning job to Azure.",
+              "usage": "azd ai finetuning deploy"
+            }
+          ],
+          "artifacts": {
+            "darwin/amd64": {
+              "checksum": {
+                "algorithm": "sha256",
+                "value": "af11047544db004c13d0c68b781fb57edeb6358656b1cf3b4b1bf80e6dffd13b"
+              },
+              "entryPoint": "azure-ai-finetune-darwin-amd64",
+              "url": "https://github.com/achauhan-scc/azure-dev/releases/download/azd-ext-azure-ai-finetune_0.0.2-preview/azure-ai-finetune-darwin-amd64.zip"
```

Comment:
if the plan is to move the code to azure-dev repo, let's hold on updating the registry until the binaries are not provided from a fork